File: src/test/resources/driver/testPascalRun/if1Test.pas
(*
 * Project: pascal_test
 * User: Lenovo
 * Date: 21/02/2022
 *)
{$mode iso}
program if1Test;
var intvar:Integer;
begin
  intvar := 9;
  if intvar> 10 then
    WriteLn('greater than 10')
  else writeln('<=10');
end.

File: src/main/java/example/if1Test.java
package example;

public class if1Test {
    private static int intvar;
    public static void main(String[] args) {
        intvar = 9;
        double d1 = 10;
        double d2 = 10;


        System.out.println(intvar > 10);

        boolean b = intvar >= 10;
        if (b) {
            System.out.println("greater than 10");
        } else {
            System.out.println("<10");
        }

    }
}
